analyzers/src/SonarAnalyzer.CSharp/Rules/TooManyParameters.cs

```diff
@@ -38,6 +38,12 @@ public class TooManyParameters : TooManyParametersBase<SyntaxKind, ParameterList
             { SyntaxKindEx.LocalFunctionStatement, "Local function" }
         }.ToImmutableDictionary();
 
+        protected override bool IsZeroOverheadMemberAccess(SyntaxNode node) =>
+            node is MethodDeclarationSyntax { AttributeLists: { Count: > 0 } attributeList } methodDeclaration
+            && methodDeclaration.Modifiers.Any(SyntaxKind.ExternKeyword)
+            && methodDeclaration.Modifiers.Any(SyntaxKind.StaticKeyword)
+            && attributeList.Any(x => x.Attributes.Any(y => y.Name.ToString() == "UnsafeAccessor"));
```

Comment:
Use `AttributeListSyntaxExtensions.GetAttributes`

Reply:
Not checking attributes anymore